content/doc/book/installing/linux.adoc

```diff
@@ -58,27 +73,28 @@ sudo apt-get update
 sudo apt-get install jenkins
 ----
 
-[NOTE]
-====
-If an error is reported, "`jenkins : Depends: daemon but it is not installable`", add the link:https://help.ubuntu.com/community/Repositories/Ubuntu["universe" apt repository] of community maintained free and open source software for Ubuntu by executing this command after `sudo apt-get update`:
-[source,bash]
-----
-sudo add-apt-repository universe
-----
-====
+Beginning with Jenkins 2.335, the weekly package is configured with systemd rather than the older System V init.
+See the link:https://www.digitalocean.com/community/tutorials/how-to-use-systemctl-to-manage-systemd-services-and-units[DigitalOcean community systemd tutorial] to better understand the benefits of `systemd` and the `systemctl` command.
 
-This package installation will:
+The weekly package installation will:
 
-* Setup Jenkins as a daemon launched on start. See `/etc/init.d/jenkins` for more details.
+* Setup Jenkins as a daemon launched on start. See `/lib/systemd/system/jenkins.service` for more details.
```

Comment:
It may also be useful to be aware of `systemctl cat jenkins.service`

Reply:
If that was an implicit suggestion to replace references to `/lib/systemd/system/jenkins.service` with references to `systemctl cat jenkins`, then message received. I have done so in commit 876f22e0f. It may also be useful to be aware of the GitHub [`suggestion`](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) block, which allows one to make explicit suggestions.

Reply:
I'm well aware of it, but wasn't quite sure on the phrasing.